source/math/FloatPoint3D.ooc

```diff
@@ -67,7 +67,7 @@ FloatPoint3D: cover {
 	operator <= (other: This) -> Bool { this x <= other x && this y <= other y && this z <= other z }
 	operator >= (other: This) -> Bool { this x >= other x && this y >= other y && this z >= other z }
 	operator as -> String { this toString() }
-	toString: func -> String { "#{this x toString()}, #{this y toString()}, #{this z toString()}" }
+	toString: func -> String { "#{"%.8f" format(this x)}, #{"%.8f" format(this y)}, #{"%.8f" format(this z)}" }
```

Comment:
That shouldn't be the main focus of toString()s, since we mainly use it for debugging. Two zeroes is not enough for everyday use for me so i have to write huge println lines all the time. And thats annoying.